src/Card.jsx

```diff
@@ -16,18 +16,21 @@ export default function Card({
 
   const answerClass = isAnswerVisible ? 'Card__answer' : 'Card__answer hidden'
 
+  const tagsWithIds = tags.map(tag => {
+    return JSON.parse(`{ "value": "${tag}", "id": "${uuidv4()}" }`)
+  })
```

Comment:
You can use the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to get everything that was previous stored in `tag` in an object and add a new property `id`. 
```suggestion
  const tagsWithIds = tags.map(tag => {...tag, id: uuidv4()})
```